exit

```diff
@@ -0,0 +1,602 @@
+[33mcommit 4ab665063c1db25b4a619ffda930e474f424fe0d[m[33m ([m[1;36mHEAD -> [m[1;32mstep[m[33m)[m
```

Comment:
What is this file? You always seem to add weird files to your commits. 
Seems like you echoed the git log into a file somehow..

Reply:
I don't know.. but it doesn't apear in my directory anymore so I guess it got deleted since.. 
I will pay more attention to weird files showing up